src/Microsoft.Azure.SignalR.AspNet/Infrastructure/ServiceConnection.cs

```diff
@@ -17,6 +17,7 @@ namespace Microsoft.Azure.SignalR.AspNet
 {
     internal partial class ServiceConnection : ServiceConnectionBase
     {
+        private const string ReconnectMessage = "{\"Reconnect\":true}";
```

Comment:
This makes me semi nervous that we aren't parsing the message

Reply:
Feels no need to ...though

Reply:
Hopefully nobody changes the spacing?

Reply:
they shouldn't. Parsing to Jason is too costly.....